easybuild/easyconfigs/q/Qhull/Qhull-2015.2-intel-2016a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'Qhull'
+version = '2015.2'
+
+homepage = 'http://www.qhull.org'
+description = """
+Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point,
+furthest-site Delaunay triangulation, and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d,
+and higher dimensions. Qhull implements the Quickhull algorithm for computing the convex hull.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+# Recently changed the Unix version numbering see http://www.qhull.org/src/Changes.txt
+# and https://github.com/Homebrew/homebrew-science/issues/3353
+sources = ['%(namelower)s-2015-src-7.2.0.tgz']
```

Comment:
Any idea if there's a relation to `2015.2` and `7.2.0`?

I'm just worried that we'll forget updating this is we bump the version...

Should we use this?

```python
sources = ['%(namelower)s-%(version_major)s-src-7.%(version_minor)s.0.tgz']
```

I'm making assumptions here, but at least we'll catch that we need to update this accordingly with a future version bump?